docker/sequencErr/1.0.0/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi7/python-36 AS builder
+
+ARG GITHUB_TOKEN
+RUN git clone --depth 1 --branch v2.0.7 https://${GITHUB_TOKEN}@github.com/stjude/sequencErr.git
```

Comment:
I noticed this is failing. Are we missing the GITHUB_TOKEN secret?

Reply:
I don't think Clay has added it yet. I've pushed the images manually.

Reply:
@claymcleod - Can you add GITHUB_TOKEN to this repo so that it can clone the SequenErr project?